src/main.rs

```diff
@@ -21,6 +21,11 @@ fn main() {
                 .about("Builds the current directory")
                 .version(env!("CARGO_PKG_VERSION")),
         )
+        .subcommand(
+            clap::SubCommand::with_name("sync")
+                .about("Creates history of current directory")
```

Comment:
It should say something like this: `sync` with `fpm-repo` or `.history` folder if not using `fpm-repo`.